test/js-native-api/test_typedarray/test.js

```diff
@@ -41,8 +41,8 @@ assert.strictEqual(externalResult[2], 2);
 // Validate creation of all kinds of TypedArrays
 const buffer = new ArrayBuffer(128);
 const arrayTypes = [ Int8Array, Uint8Array, Uint8ClampedArray, Int16Array,
-                     Uint16Array, Int32Array, Uint32Array, Float32Array,
-                     Float64Array, BigInt64Array, BigUint64Array ];
+                     Uint16Array, Int32Array, Uint32Array, Float16Array,
```

Comment:
Note that the unit test is failing because the test `binding.gyp` must be changed to use the `NAPI_EXPERIMENTAL` macro. Otherwise, it is compiled with the latest stable Node-API version.
See: https://github.com/nodejs/node/blob/c08a1d152b758a1b97b81e9edce6ed60faaf4063/test/js-native-api/test_finalizer/binding.gyp#L5 as example.